layouts/blog/single.html

```diff
@@ -28,6 +28,21 @@
             <div class="disqus">
                 {{ template "_internal/disqus.html" . }}
             </div>
+            {{ if .Site.Params.gitalk }}
+            <div id="gitalk-container"></div>
+              <script type="text/javascript">
+                const gitalk = new Gitalk({
+                    clientID: {{ .Site.Params.gitalk_clientID }},
```

Comment:
should the value be inside quotes so that it becomes a JavaScript string? (here and below)

```
clientID: "{{ .Site.Params.gitalk_clientID }}",
```

Reply:
I've tested this typo with no error. But it could be nicer to add a double quote in case of confusion.